source/modules/connection_manager.py

```diff
@@ -47,8 +51,9 @@ def __init__(self, version: Version, proxy_type=None) -> None:
         self.manager: REQUEST_MANAGER | None = None
 
         # Basic Headers
-        self._headers = {"user-agent": f"Blender-Launcher-v2/{self.version!s} ({get_platform_full()})"}
-
+        # TODO: Change back BLV2 to Blender-Launcher-v2 when it will be unblocked
```

Comment:
Do we plan to apply this before or after the merge?

Reply:
Oh yeah, I forgot to swap it back. Good thing I added the comment!